### PR TITLE
Self-host ClojureScript port

### PR DESCRIPTION
**Pull request by arichiardi**
This is a twin patch for honeysql and honeysql-postgres, adding support for
ClojureScript self-host. It also introduces tach for testing (lein tach lumo).

I have bumped the version of honeysql to an arbitrary one but the patch in
there needs to happen first

### CODE REVIEW

**Comment by arichiardi** [comment]
ping

File: project.clj [hunk 1]
```diff
@@ -4,4 +4,7 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [honeysql "0.6.3"]])
+                 [honeysql "0.9.2"]]
```
Comment (ukriish) [inline project.clj:7]: The latest available version of honeysql is 0.9.1, please fix this.

**Review by ukriish — COMMENTED** [review]
@arichiardi Sorry about the delay in getting back, somehow these got lost in my mails.

I've added a comment. Can you please check that?

File: project.clj (hunk 1 above)
Comment (arichiardi) [reply project.clj:7]: Yes I can fix it, however I think that this patch does not make sense without the other in `honeysql`, so we might need to wait for that to be released (has been merged)

Comment (ukriish) [reply project.clj:7]: Okay. Will consider this once that 0.9.2 has been released. 

Also, thanks for raising the PR.

Comment (arichiardi) [reply project.clj:7]: @krish8664 honeysql got released :tada: :smile:

Comment (ukriish) [reply project.clj:7]: @arichiardi Finally :), I'll merge this by Monday.

Comment (arichiardi) [reply project.clj:7]: @krish8664 Great! I also created an `npm` org where we can deploy this package as well. If you have a login/username I can add you or I can publish myself, up to you :smile: 

https://www.npmjs.com/settings/honeysql/packages

Comment (arichiardi) [reply project.clj:7]: ping :+1: :smile: :100:

Comment (ukriish) [reply project.clj:7]: ah. sorry. merging this. 

You can publish the npm package 👍

**Pull request merged**